### DISCUSSION TITLE
Nodal Stress Value

### DISCUSSION THREAD

**Discussion by danishTUE** [neperfepx/FEPX#65] [Q&A] [answered]
Hello, I have recently started using FEPX. As I understand, one can get elemental level output of stress tensor (which I presume means is integration point level output?). Is it possible to get nodal stress output values as well? thanks in advance!

**Comment by mkasemer** [comment, marked as answer]
@danishTUE 

Please refer to the manual for possible output options.

To answer your question directly, no, we do not output stresses at the nodal points, as the stress is not calculated at the nodal points.

The stress is calculated at the quadrature points of the element. We consider 15 quadrature points in 10-node quadratic elements. Since the gradient of stress across a single element is rather small, and to reduce the size of output (which would increase 15-fold if we printed the stress at every quad point), we print the stress at only a single quad point. In other words: it isn't worth it to print such a large set of data, as it rarely affords the user necessary information of the deformation field (this goes for all variables printed elementally). 

For the quadrature rule that we utilize, one of the quad points lies at the centroid of the element, and this is the value that we print.
